WooCommerce/Classes/Extensions/UIView+SubviewsAxis.swift

```diff
@@ -5,7 +5,7 @@ extension UIView {
     /// This is based on the assumption that there are two subviews.
     /// Otherwise, nil is returned.
     ///
-    func axisOfTwoSubviews() -> UIView.Axis? {
+    func axisOfTwoSubviews() -> NSLayoutConstraint.Axis? {
```

Comment:
unrelated: this doesn't seem to be used in the codebase anymore, we can delete this at some point 🙂

Reply:
Opened #5957